### ISSUE TITLE
V0.7.1 release for Darwin 386 is an empty TAR file

### ISSUE THREAD

**Issue by apatrida** [cycloidio/terracognita#212]
The release: https://github.com/cycloidio/terracognita/releases/tag/v0.7.1
has 120MB+ files for everything other than Darwin 386 which is `45 Bytes` and an empty tar.

Something wrong with this release?

**Comment by xescugc** [comment]
Yes you are right and for what I see we have been dragging it for a while, I'm checking it now to solve it.
